### PR TITLE
fix: core cleanup

### PR DESCRIPTION
**Pull request by coryrylan**
## PR Checklist

Please check if your PR fulfills the following requirements:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [x] Docs have been added / updated (for bug fixes / features)
* [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

* [x] Bugfix
* [ ] Feature
* [ ] Code style update (formatting, local variables)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [x] Documentation content changes
* [ ] clarity.design website / infrastructure changes
* [ ] Other... Please describe:

## What is the new behavior?
- alerts icon issue
- changelog for core
- remove core from component status
- broken links in core getting started

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

### CODE REVIEW

File: src/clr-core/.storybook/public/demo.css [hunk 1]
```diff
@@ -66,6 +66,16 @@ p {
   margin-left: 2rem;
 }
 
+.doc .link-list {
```
Comment (coryrylan) [inline src/clr-core/.storybook/public/demo.css]: will clean this work up once we have typography/core components worked out

File: src/clr-core/alert/alert.base.ts [hunk 2]
```diff
@@ -24,6 +25,8 @@ import {
 } from '@clr/core/icon-shapes';
 import { html, LitElement } from 'lit-element';
 
+ClarityIcons.addIcons(checkCircleIcon, infoCircleIcon, exclamationCircleIcon, exclamationTriangleIcon, timesIcon);
```
Comment (coryrylan) [inline src/clr-core/alert/alert.base.ts]: The missing icon import and registering icons late was causing them to not be visible if you only imported alerts.

Comment (mathisscott) [reply src/clr-core/alert/alert.base.ts]: Interesting. Yeah, this was the kind of stuff I had to do in the sample apps. It looks like it's going to be de-riguer for teams using the icons in a tree-shakeable manner.

File: src/clr-core/welcome.stories.ts [hunk 3]
```diff
@@ -124,15 +124,17 @@ export const gettingStarted = () => {
 
 <!-- or use CSS directly -->
 <link href="/node_modules/@clr/core/global.min.css" rel="stylesheet">
-<link href="/node_modules/@clr/city/css/bundles/default.min" rel="stylesheet">
+<link href="/node_modules/@clr/city/css/bundles/default.min.css" rel="stylesheet">
```
Comment (mathisscott) [inline src/clr-core/welcome.stories.ts]: Good catch! 👍

**Pull request merged**